curriculum/challenges/espanol/blocks/review-react-state-and-hooks/67c2bbd476f6e540b4c0b0d4.md

```diff
@@ -51,8 +51,8 @@ function handleDelete(id) {
 
 ## Trabajando con Estado y el Hook `useState`
 
-- **Definición de estado**: En React, el estado es un objeto que contiene datos para un componente. Cuando el estado se actualiza, el componente se volverá a renderizar. React trata el estado como inmutable, lo que significa que no debe modificarlo directamente.
-- **Hook `useState()`**: El hook `useState` es una función que te permite declarar variables de estado en componentes funcionales. Aquí tienes una sintaxis básica:
+- <strong x-id= "1">Definición de estado</strong>: En React, el estado es un objeto que contiene datos para un componente. Cuando el estado se actualiza, el componente se volverá a renderizar. React trata el estado como inmutable, lo que significa que no debe modificarlo directamente.
```

Comment:
I don't think this change is correct/needed.

Reply:
Solved in the next update. Thanks for checking that.